cfgov/v1/admin.py

```diff
@@ -12,7 +13,12 @@
 
 @admin.register(User)
 class UserAdmin(UserAdmin):
-    actions = UserAdmin.actions + ["send_password_reset_email"]
+    if django.VERSION >= (4, 1, 0):
```

Comment:
```suggestion
    if django.VERSION >= (4, 1):
```

Not to be extra pedantic, but

```
>>> (4, 1) >= (4,1,0)
False
>>> (4, 1) >= (4,1)
True
>>> (4, 1, 0) >= (4, 1)
True
>>> (4, 1, 1) >= (4, 1)
True
```

Reply:
Done in [972d735](https://github.com/cfpb/consumerfinance.gov/pull/8188/commits/972d735c1496ef98422295f5495f6f04a4e1c164)